yarn.lock

```diff
@@ -187,6 +187,13 @@
   version "3.5.1"
   resolved "https://registry.yarnpkg.com/@types/diff/-/diff-3.5.1.tgz#30253f6e177564ad7da707b1ebe46d3eade71706"
 
+"@types/electron-store@^1.3.1":
```

Comment:
Tried opening a CQ last friday but for some reason the form sending failed on the eclipse website, so I kind of gave up at that moment. But checking the license manually it is MIT so it should be fine.

Should I open a CQ nonetheless? @marcdumais-work

Reply:
> Should I open a CQ nonetheless? @marcdumais-work

Yes, please try again. This is a dev-dependency so you can follow [these instructions](https://github.com/theia-ide/theia/wiki/Registering-CQs#case-new-or-updated-testbuild-dependency-eg-devdependencies).

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19539

Reply:
> https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19539

You need to clearly specify that this is a built/test dependency and also request that this CQ covers from a specific version of the dependency as well as later versions. IOW, follow the instructions from the link above :)

Reply:
The CQ is marked as PMC approved, what does that mean?

Reply:
Someone from the PMC has voted on the CQ already. Not sure if they understood the context or not, but I have added some info.

Reply:
CQ looks ok